### PR TITLE
Update dolphin-emu.spec

### PR DESCRIPTION
**Pull request by KAMiKAZOW**
Update to adapt to Dolphin changes and fix some oversights:
* Installs udev rules
* More generic spec file that uses name variables more to simplify testing of other branches
* Adaptations for Qt GUI, Cubeb sound back-end, and Dolphin's "or later" licensing provision
* %post / %postun scripts

### CODE REVIEW

**Comment by spycrab** [comment]
@dolphin-emu-bot rebuild

File: Installer/dolphin-emu.spec [hunk 1]
```diff
@@ -15,133 +15,200 @@
...
-# Disable miniupnpc in OBS for F20
-BuildRequires:  miniupnpc-devel
-BuildRequires:  openal-soft-devel
-#polarssl is now mbedtls:
-BuildRequires:  mbedtls-devel
-BuildRequires:  SDL2-devel
-BuildRequires:  SFML-devel
-BuildRequires:  SOIL-devel
-BuildRequires:  soundtouch-devel
+BuildRequires:    cmake >= 2.8
```
Comment (leoetlino) [inline Installer/dolphin-emu.spec]: We require cmake 3.5 (or something like that).

**Comment by KAMiKAZOW** [comment]
> Could you squash your commits please?

No, I can't. You are the guys with the green Squash&Accept button. I don't use git.

**Comment by leoetlino** [comment]
We don't use GitHub's built-in squash & merge feature, because that gets rid of the PR merge commits (history).

**Comment by KAMiKAZOW** [comment]
I still don't use git. I have no clue how to use git. Feel free to simply pick the dolphin-emu.spec file from my home repo if that makes things easier for you.

**Comment by leoetlino** [comment]
Could you tick the "Allow edits from maintainers" checkbox so that people can fix the branch for you?

**Comment by KAMiKAZOW** [comment]
It is ticked…

**Comment by leoetlino** [comment]
Well, then you'll have to rebase this on your own because GitHub won't let me push changes to your branch. Learning git is not that complicated. Feel free to open a new PR after the history has been fixed.

**Pull request closed without merging**